tests/server/test_compilerservice.py

```diff
@@ -840,45 +840,22 @@ async def old_compile_report(server_with_frequent_cleanups, environment_for_clea
 async def test_compileservice_cleanup(
     server_with_frequent_cleanups, client_for_cleanup, environment_for_cleanup, old_compile_report
 ):
-    # There is a new and an older report in the database
```

Comment:
1. Would it not be better to set 'config.Config.set("server", "compiler-report-retention", "2")' to a higher value, so we still test that not too much is cleaned up
2. it may help to add a comment at the start of the testcase explaining there is an old and a new report, one should be cleaned up within a second.

Reply:
Good idea. I will implement the changes.